core/domain/exp_services.py

```diff
@@ -1886,15 +1886,16 @@ def validate_exploration_for_story(
         if state.interaction.id == 'MultipleChoiceInput':
             choices = (
                 state.interaction.customization_args['choices'].value)
-            error_string = (
-                'Exploration in a story having MultipleChoiceInput '
-                'interaction should have at least 4 choices present. '
-                'Exploration with ID %s and state name %s have fewer than '
-                '4 choices.' % (exp.id, state_name)
-            )
-            if len(choices) < 4 and strict:
-                raise utils.ValidationError(error_string)
-            validation_error_messages.append(error_string)
```

Comment:
**Note:** This old code was buggy. I fixed it in this PR.

Reply:
Do we need to test the changes to this validation logic on the server?

Reply:
Should be good without testing. All multiple choice questions in curated explorations have 4 choices. And if anything, we would have run into "too many" validation errors rather than "too few".